Cython/Compiler/Code.py

```diff
@@ -2469,6 +2538,18 @@ def put(self, code):
         elif fix_indent:
             self.level += 1
 
+    def put_and_process_code_here(self, code_str: str):
+        class DummyUtilityCode:
+            name = "<unamed>"
+            def format_code(self, code):
+                return code
```

Comment:
This doesn't seem great, but it was difficult to find another way to process utility code loaded with `load_as_string`.

An alternative might be something like in https://github.com/cython/cython/pull/5799/files#diff-1c3b7395c96b4ecec571dc9f3fb5f1689faa4be6006abe7c3f01145f1010cf9c ?

Reply:
... and it doesn't work anyway when compiled since Cython type-checks the arguments.